pkg/monitoring/iot.go

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package monitoring
+
+import (
+	"context"
+	iotv1alpha1 "github.com/enmasseproject/enmasse/pkg/apis/iot/v1alpha1"
+	custom_metrics "github.com/enmasseproject/enmasse/pkg/custom-metrics"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"time"
+)
+import logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+var log = logf.Log.WithName("monitoring")
+
+func StartIoTMetrics(mgr manager.Manager) {
```

Comment:
@ctron @robshelly Just thinking about Monitoring CRD work for a moment. Would this logic likely be moved into the monitoring_controller for the monitoring CRD?